target/linux/ar71xx/image/generic.mk

```diff
@@ -801,6 +801,20 @@ define Device/jwap230
 endef
 TARGET_DEVICES += jwap230
 
+define Device/k2t
+  DEVICE_TITLE := Phicomm K2T A1/A2/A3 board (16MB flash)
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-storage kmod-ath10k ath10k-firmware-qca9888
+  BOARDNAME := K2T
+  IMAGE_SIZE := 15744k
+  MTDPARTS := spi0.0:192k(u-boot)ro,64k(config)ro,320k(permanent),15744k(firmware),64k(art)ro
+  SUPPORTED_DEVICES := k2t
+  KERNEL_INITRAMFS := kernel-bin | patch-cmdline | lzma | uImage lzma
+  KERNEL_INITRAMFS_SUFFIX := -factory.bin
```

Comment:
Is the initramfs factory firmware created here useless?
I think there shouldn't be a factory.bin suffix if it's not designed for flashing the device from manufacturer's firmware.